src/pages/login/test.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import LoginPage, { Copyright } from './index';
+
+jest.mock('react-router-dom', () => ({
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['jest' is not defined.](https://app.codacy.com/manual/iagoregiani/verificador-associacao/pullRequest?prid=5497764)